protos/netstate/netstate.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+package netstate;
+
+// NetState defines the interface for interacting with the network state persistence layer
+service NetState {
+  // Put formats and hands off a file path to be saved to boltdb
+  rpc Put(FilePath) returns (PutResponse);
+  // Get formats and hands off a file path to get from boltdb
+  rpc Get(FilePath) returns (GetResponse);
+  // List calls the bolt client's List function and returns all file paths
+  rpc List(ListRequest) returns (ListResponse);
+  // Delete formats and hands off a file path to delete from boltdb
+  rpc Delete(FilePath) returns (DeleteResponse);
+}
+
+// FilePath is a request message for Put, Get, and Delete rpc calls
+message FilePath {
+  string path = 1;
+  // smallValue is a value too small to be broken up and stored
+  // in different places
+  string smallValue = 2;
+}
+
+// ListRequest is a request message for the List rpc call
+message ListRequest {
+  string request = 1;
```

Comment:
what is request supposed to be ?

Reply:
It's just a request from the client to list all of the file paths saved in the db. I wasn't planning on `List()` taking any arguments initially, but protoc didn't like that..

Reply:
ah ok this can just be
```proto 
message ListRequest {}
```

you don't have to define anything inside the message

Reply:
Oh nice, thanks!